eclipse/che-theia
# Align built-in extensions version with upstream

### What does this PR do?
Backport of https://github.com/eclipse-theia/theia/pull/9017 to fix https://github.com/eclipse-theia/theia/issues/9014 in Che-Theia assembly.

Signed-off-by: Vladyslav Zhukovskyi <vzhukovs@redhat.com>


### Screenshot/screencast of this PR
N/A


### What issues does this PR fix or reference?
N/A


### How to test this PR?
1. Build Che-Theia from `build.sh` scripts in `/dockerfile` directory.
Confirm that build successful.

2. Confirm that Github PR checks are successful (build steps).


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable
